packages/flutter/lib/src/gestures/constants.dart

```diff
@@ -54,6 +54,11 @@ const Duration kZoomControlsTimeout = Duration(milliseconds: 3000);
 /// the gesture is a scroll gesture, or, inversely, the maximum distance that a
 /// touch can travel before the framework becomes confident that it is not a
 /// tap.
+///
+/// Mathematically speaking, if a touch moves with a total delta in the range
+/// [0, 'kTouchSlop'] it is not considered to be a drag, whereas if the total
+/// delta is in the range ('kTouchSlop', Infinity), it is considered to be a
```

Comment:
this is probably clearer if written as "... a total delta less than or equal to [kTouchSlop] it is not considered to be a drag, whereas if the total delta is greater than [kTouchSlop], it is considered..."

That also avoids any risk that dartdoc will get confused by the `[...]`